### PR TITLE
refactor: removed JQuery dependency in Verilog2CV.js file

### PR DESCRIPTION
**Pull request by disha1202**
Fixes #27

### CODE REVIEW

**Comment by devartstar** [comment]
@disha1202  sorry for the late response.
I will look into your PRs during the weekend.

File: src/simulator/src/Verilog2CV.js [hunk 1]
```diff
@@ -65,23 +65,23 @@ export function verilogModeSet(mode) {
...
-        $('#code-window').show()
-        $('.elementPanel').hide()
-        $('.timing-diagram-panel').hide()
-        $('.quick-btn').hide()
-        $('#verilogEditorPanel').show()
+        document.querySelector('#code-window').style.display = 'block';
+        document.querySelector('.elementPanel').style.display = 'none';
+        document.querySelector('.timing-diagram-panel').style.display = 'none';
+        document.querySelector('.quick-btn').style.display = 'none';
+        document.querySelector('#verilogEditorPanel').style.display = 'block';
```
Comment (devartstar) [inline src/simulator/src/Verilog2CV.js:72]: lets use `getElementById` for ids, since its faster then `querySelector`

Comment (disha1202) [reply src/simulator/src/Verilog2CV.js]: Thanks for reviewing!
I have replaced querySelector by getElementById.

Comment (devartstar) [reply src/simulator/src/Verilog2CV.js]: cool :100:

**Pull request merged**